packages/cli/src/commands/deploy/index.ts

```diff
@@ -195,6 +195,13 @@ export default async (client: Client) => {
   let sourceFilesOutsideRootDirectory: boolean | undefined = true;
 
   if (status === 'not_linked') {
+    if (localConfig?.projectSettings) {
+      output.error(
+        'New projects cannot contain `projectSettings` in vercel.json'
```

Comment:
This error message is a bit confusing. Why can't they? Perhaps we should have a "More info" link that explains why this is the case and what the user should do about it.

Reply:
Ill add a link to the incoming projectSettings section and then expand on the reasoning there